catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/OpenSearchSource.java

```diff
@@ -196,18 +193,22 @@ public void destroy() {
         LOGGER.info("Nothing to destroy.");
     }
 
-    protected void configureClient() {
-        openSearchConnection = new OpenSearchConnection(endpointUrl, filterAdapter,
-                securitySettingsService, username, password);
-    }
-
     @Override
     public boolean isAvailable() {
         boolean isAvailable = false;
         if (!lastAvailable || (lastAvailableDate
                 .before(new Date(System.currentTimeMillis() - AVAILABLE_TIMEOUT_CHECK)))) {
 
-            WebClient client = openSearchConnection.getOpenSearchWebClient();
+            if (factory == null) {
+                return false;
+            }
+
+            WebClient client = null;
+            try {
+                client = factory.getUnsecuredWebClient();
```

Comment:
You would also need to use basic auth here if a username and password were provided, otherwise the client will error out. I would recommend setting up the opensearch endpoint with basic auth and then doing loopback federation to make sure that basic auth works.